src/flamenco/features/fd_features.h

```diff
@@ -14,6 +14,11 @@
 
 #define FD_FEATURE_DISABLED (ULONG_MAX)
 
+/* FD_FEATURES_CLUSTER_VERSION is the value that will be used to
+   determine the cluster version when turning on features.
+*/
+#define FD_FEATURES_CLUSTER_VERSION (2000)
```

Comment:
Could we make it clear this a default, and move it somewhere out of the features-specific code? Since it's not to do with features, but is the default cluster version for the whole app. It should probably be defined close to where the cluster version is set.

Reply:
makes sense, updated